### PR TITLE
Added E2E for #114

### PR DESCRIPTION
**Pull request by Den1552**
## Summary

Added E2E for #114. 
Creating new TST that imports an coded test. Check in E2E if we can edit the coded test with `// vmock` autocompletion. 

## Added
`vcast_coded_tests_relative_path.test.ts` E2E test.
Added `import_coded_tests` spec group.

### CODE REVIEW

File: tests/internal/e2e/test/specs/vcast_coded_tests_relative_path.test.ts [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+    const workspaceFolderSection =
+      await expandWorkspaceFolderSectionInExplorer("vcastTutorial");
+
+    const configFile = await workspaceFolderSection.findItem("CCAST_.CFG");
+    await configFile.openContextMenu();
+    await (await $("aria/Set as VectorCAST Configuration File")).click();
+  });
+
+  // it("should create VectorCAST environment", async () => {
```
Comment (Zbigor) [inline tests/internal/e2e/test/specs/vcast_coded_tests_relative_path.test.ts:106]: If this is not used, we should get rid of the commented out code

File: tests/internal/e2e/test/specs_config.ts [hunk 2]
```diff
@@ -119,6 +119,12 @@ export function getSpecGroups(useVcast24: boolean) {
       env: {},
       params: {},
     };
+
```
Comment (Zbigor) [inline tests/internal/e2e/test/specs_config.ts:122]: I have a feeling you have a change in `wdio.conf.ts` that you haven't commited based on this

File: tests/internal/e2e/test/specs_config.ts [hunk 3]
```diff
@@ -119,6 +119,12 @@ export function getSpecGroups(useVcast24: boolean) {
...
+    specGroups["import_coded_test"] = {
+      specs: ["./**/**/vcast_coded_tests_relative_path.test.ts"],
+      env: { IMPORT_CODED_TEST_IN_TST: "True" },
```
Comment (Zbigor) [inline tests/internal/e2e/test/specs_config.ts:125]: Where is `IMPORT_CODED_TEST_IN_TST` used?

File: tests/internal/e2e/test/specs_config.ts (hunk 2 above)
Comment (Zbigor) [reply tests/internal/e2e/test/specs_config.ts:122]: Now it's in

File: tests/internal/e2e/test/specs_config.ts (hunk 3 above)
Comment (Den1552) [reply tests/internal/e2e/test/specs_config.ts:125]: Here, to determine what env setup should be prepared.

https://github.com/vectorgrp/vector-vscode-vcast/blob/769b336424c0d3e499a534b0c323ea1fa7a06718/tests/internal/e2e/test/wdio.conf.ts#L323-L338

**Pull request merged**